script/import_scripts/bbpress.rb

```diff
@@ -195,6 +229,48 @@ def import_topics_and_posts
     end
   end
 
+  def create_permalinks
+    puts '', 'Creating redirects...', ''
+
+    puts '', 'Users...', ''
+    # https://staging-gemsociety.kinsta.com/members/m76steve/
+    User.find_each do |u|
+      ucf = u.custom_fields
+      next unless ucf["import_id"]
+      if ucf && ucf["import_id"] && ucf["import_username"]
+        Permalink.create( url: "/members/#{ucf['import_username']}", external_url: "/users/#{u.username}" ) rescue nil
+        print '.'
+      end
+    end
+
+    puts '', 'Topics...', ''
+    Post.find_each do |post|
+      pcf = post.custom_fields
+      if pcf && pcf["import_id"]
+        topic = post.topic
+        id = pcf["import_id"].split('#').last
+        if post.post_number == 1
+          slug = Slug.for(topic.title)
+          Permalink.create( url: "#{BASE}topic/#{slug}", topic_id: topic.id ) rescue nil
+        end
+        print '.'
+      end
+    end
+
+    puts '', 'Categories...', ''
+    Category.find_each do |cat|
+      ccf = cat.custom_fields
+      next unless id = ccf["import_id"]
```

Comment:
You're right. I improved the error checking (rescuing failed DB connection) and removed that, but now I can't figure out how to push the stuff back because of your commit.

My inclination is to delete this branch and just try again, as I can't figure out how to pull your commit back into my fork.